### ISSUE TITLE
Stop working after resend message with photo

### ISSUE THREAD

**Issue by lupin-de-mid** [soberhacker/obsidian-telegram-sync#77]
- installed plugin via BRAT
- setup
- tested ok  with text messages
- forward message with photo
- photo appeared on disk
- no message file appeared on disk
- message in chat not removed
- no other messages processed after that moment
- no messages in log
- try restart of  obsidian ( with enabled and disabled plugin) no luck

- if I disable plugin and reenable without restart  error appear 
```Error: ETELEGRAM: 409 Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
    at eval (plugin:telegram-sync:60565:17)
From previous event:
    at Promise2.longStackTracesCaptureStackTrace [as _captureStackTrace] (plugin:telegram-sync:6737:27)
    at Promise2._then (plugin:telegram-sync:8516:23)
    at Promise2.then (plugin:telegram-sync:8430:25)
    at Promise2.caught.Promise2.catch (plugin:telegram-sync:8411:25)
    at TelegramBotPolling._getUpdates (plugin:telegram-sync:60282:62)
    at TelegramBotPolling._polling (plugin:telegram-sync:60219:34)
    at eval (plugin:telegram-sync:60262:58)
```

or this  (that start from constructor)

```
Error: ETELEGRAM: 409 Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
    at eval (plugin:telegram-sync:60565:17)
From previous event:
    at Promise2.longStackTracesCaptureStackTrace [as _captureStackTrace] (plugin:telegram-sync:6737:27)
    at Promise2._then (plugin:telegram-sync:8516:23)
    at Promise2.then (plugin:telegram-sync:8430:25)
    at Promise2.caught.Promise2.catch (plugin:telegram-sync:8411:25)
    at TelegramBotPolling._getUpdates (plugin:telegram-sync:60282:62)
    at TelegramBotPolling._polling (plugin:telegram-sync:60219:34)
    at TelegramBotPolling.start (plugin:telegram-sync:60170:21)
    at TelegramBot2.startPolling (plugin:telegram-sync:60660:30)
    at new TelegramBot2 (plugin:telegram-sync:60459:18)
    at TelegramSyncPlugin.initTelegramBot (plugin:telegram-sync:66284:16)
    at TelegramSyncPlugin.onload (plugin:telegram-sync:66222:16)
```

**Comment by lupin-de-mid** [comment]
So I switched to #76 amd observe message in log 
about arrived message
i turned off AppendAll to Telegram.md amd after that new messages appeared as separate file, but appending to one file still not working

**Comment by lupin-de-mid** [comment]
So I found that I use location as `/input`
So obsidian  return null in this code , but unable to create this file after that, and fail without logs
```
const telegramMdPath = location ? `${location}/Telegram.md` : 'Telegram.md';
let telegramMdFile = this.app.vault.getAbstractFileByPath(telegramMdPath) as TFile;
```

**Comment by soberhacker** [comment]
Hi @lupin-de-mid 
Thanks for powerful report about this bug!
Please, check resolving this bug in the new release [Telegram Sync 1.2.0](https://github.com/soberhacker/obsidian-telegram-sync/releases/tag/1.2.0):
- add possibility to run plugin on a few desktops
- add displaying errors
- add forwardFrom to template parsing
- access allowed only for selected telegram users
- improve bot shutdown behavior when plugin is disabled and enabled again
- fix bug with non existed folder
- improve message deletion
- fix template variables names (context -> content) 

Best Regards

**Comment by soberhacker** [comment]
Fixed in release 1.3.0
https://github.com/soberhacker/obsidian-telegram-sync/releases/tag/1.3.0
